Sources/StreamChatUI/CommonViews/ContainerView.swift

```diff
@@ -5,7 +5,7 @@
 import UIKit
 
 extension UIView {
-    func flexible(axis: NSLayoutConstraint.Axis) -> Self {
+    public func flexible(axis: NSLayoutConstraint.Axis) -> Self {
```

Comment:
does this need to be public? I'd probably be a little bit surprised to find new methods on `UIView` after importing `StreamChatUI`. But maybe we do this already somewhere else. Also, if it shall stay public, it should be documented. The same comment is for `CGFloat.auto`